src/layouts/page-layout.scss

```diff
@@ -41,14 +41,43 @@
   @include carbon-content-overrides;
 }
 
-.cs--page-layout-content {
+@mixin default-project-type() {
+  h1,
+  dd {
+    @include type-style('heading-07');
+
+    margin-block-end: $spacing-08;
+  }
+
+  h3 {
+    margin-block-end: $spacing-05;
+  }
+
+  p,
+  dt {
+    margin-block-end: $spacing-05;
+    margin-inline-end: $spacing-08;
+  }
+
+  dt {
+    @include type-style('label-02');
+  }
+}
+
+:where(.cs--page-layout-content) {
+  // low specificity to make it easier to override defaults
   @include default-type;
 
+  // after Carbon default type
+  @include default-project-type;
+}
+
+.cs--page-layout-content {
   // These following styles size the content area and move the scroll
   // from the body to this content area
   block-size: calc(100vh - $spacing-09);
   max-block-size: calc(100vh - $spacing-08);
   padding-block-start: $spacing-05;
-  overflow-y: auto;
+  overflow: hidden auto;
```

Comment:
This doesn't help with #80 and even makes the situation worse because now a part of the content gets completely cut-off (see the e partially visible at the end). Maybe we can solve #80 independently? At the moment I see this as a risky change.

Reply:
80 was resolved in #127